exchange/exchange.go

```diff
@@ -798,6 +805,16 @@ func (e *exchange) getAllBids(
 		}
 		//but we need to add all bidders data to adapterExtra to have metrics and other metadata
 		adapterExtra[brw.bidder] = brw.adapterExtra
+
+		if !extraRespInfo.bidsFound && adapterBids[brw.bidder] != nil && len(adapterBids[brw.bidder].Bids) > 0 {
```

Comment:
It's not necessary to test a slice for both nil and an empty length. Please simplify to:
`!extraRespInfo.bidsFound && len(adapterBids[brw.bidder].Bids) > 0`